tutorials/holohub_operators_external_applications/README.md

```diff
@@ -439,12 +439,87 @@ cmake -DCMAKE_VERBOSE_MAKEFILE=ON ..
 
 See the `main.cpp`, `main.py`, and `CMakeLists.txt` files in this directory for complete working examples that demonstrate how to use the AJA source operator from Holohub in both C++ and Python applications.
 
+## Using Holohub CLI in External Projects
+
+The Holohub CLI provides convenient command-line tools for managing Holohub applications, including building, running, and testing operators and applications. You can easily integrate the CLI into your external projects to leverage these functionalities.
+
+### Adding holohubCLI_wrapper to Your Project
+
+To use the Holohub CLI in your external project, simply copy the `holohubCLI_wrapper` script from this tutorial to the root of your project.:
+
+```bash
+# Copy the CLI script to your project root
+cp tutorials/holohub_operators_external_applications/holohubCLI_wrapper /path/to/your/project/myprojectCLI
```

Comment:
Do we require the user to have checked out HoloHub already? Maybe we could also provide an alternate `curl` / `wget` command to fetch the raw file from GitHub without a HoloHub local clone?